price/priceService.ts

```diff
@@ -1,9 +1,23 @@
 import { baseUrl } from './url.js';
+import {Pair} from '../types/common.js'
 
-type Pair = string;
 type TimeFrame = string;
 
-async function fetchData(pair: Pair, timeFrame: TimeFrame): Promise<string> {
+interface candles {
+  startedAt: "string",
+  ticker: "string",
+  resolution: "1MIN",
```

Comment:
What about the other resolutions? This type should be a union of all valid resolutions. (Google about "union types" in Typescript.)

Reply:
okay I add other valid  resolutions.